scripts/get-tracking-version.js

```diff
@@ -1,139 +0,0 @@
-#!/usr/bin/env node
```

Comment:
@albertteoh did you delete this by mistake?  I don't am not sure what it is supposed to do, but it is being used during the build https://github.com/jaegertracing/jaeger-ui/blob/f391b862ce143ef32d60947053130c6bdee468c5/packages/jaeger-ui/package.json#L103

Reply:
https://github.com/jaegertracing/jaeger-ui/pull/1998